docs/components/banners.md

```diff
@@ -0,0 +1,252 @@
+<!--docs:
+title: "Material Banners"
+layout: detail
+section: components
+excerpt: "A banner displays a prominent message and related optional actions."
+iconId: 
+path: /catalog/material-banners/
+-->
+
+# Banners
+
+A [banner](https://material.io/components/banners) displays a prominent message and related optional actions.
+
+![Hero image of a banner showing a transaction error message](assets/banners/banners_hero.png)
+
+## Contents
```

Comment:
nit: there should only be links to 2nd level headers, and there should be a link to #contents at the top

Reply:
gotcha, done